src/jxl.imageio/jxl_pvt.h

```diff
@@ -0,0 +1,60 @@
+// Copyright Contributors to the OpenImageIO project.
+// SPDX-License-Identifier: Apache-2.0
+// https://github.com/AcademySoftwareFoundation/OpenImageIO
+
+
+/////////////////////////////////////////////////////////////////////////////
+// Private definitions internal to the jxl.imageio plugin
+/////////////////////////////////////////////////////////////////////////////
+
+
+#pragma once
+
+
+OIIO_PLUGIN_NAMESPACE_BEGIN
+
+class JxlInput final : public ImageInput {
+public:
+    JxlInput() { init(); }
+    ~JxlInput() override { close(); }
+    const char* format_name(void) const override { return "jxl"; }
+    int supports(string_view feature) const override
+    {
+        return (feature == "exif" || feature == "ioproxy");
+    }
```

Comment:
I would say that by default, the definition for JxlInput class should be in jxlinput.cpp if that's the only place that reads it.

Some of our format plugins have a format_pvt.h that contains definitions that will be needed by both the input and output classes. You may want to wait until there is an implemented output (when you will know what needs to be shared) before making an extra header for the shared definitions.